src/Enemy.cpp

```diff
@@ -57,11 +58,11 @@ void Enemy::DisplayHUD() {
 
 
 
-vector<int> Enemy::GetDrops() {
+map<ITEMTYPE, int> Enemy::GetDrops() {
     // Returns the number of items dropped when the enemy is defeated depending on the item won.
-    vector<int> drops;
+    map<ITEMTYPE, int> drops;
     for (int i = 0; i < NUM_ITEMS; i++) {
-        drops.push_back(ReturnItemDrop(i));
+        drops.insert(make_pair(static_cast<ITEMTYPE>(i + 1), ReturnItemDrop(i)));
```

Comment:
Instead of doing a cast here, maybe it's better to change `Enemy::ReturnItemDrop(int)` to take in an `ITEMTYPE` instead of an `int`. Or better yet, change `ReturnItemDrop` to `ReturnItemDrops`, and make it return a `map<ITEMTYPE, int>`.

Reply:
@tagniam done